process-services/2.4/config/authenticate.md

```diff
@@ -37,7 +37,7 @@ Configure the `activiti-identity-service.properties` file using the below proper
 |keycloak.realm| *Required.* Name of the realm configured in the Identity Service.|
 |keycloak.auth-server-url| *Required.* Base URL of the Identity Service server in the format `https://{server}:{port}/auth`|
 |keycloak.ssl-required| *Required.* Whether communication to and from the Identity Service server is over HTTPS. Possible values are `all` for all requests, `external` for external requests or `none`. **Important:** this property needs to match the equivalent setting for **Require SSL** in your realm within the Identity Service administration console.|
-|keycloak.resource| *Required.* The **Client ID** for the client created within your realm that points to Process Services.|
+|keycloak.resource| *Required.* The **Client ID** for the client created within your realm that points to Process Services. <br><br> **Note**: For Keycloak compatibility, you must enable **Exclude Issuer from Authentication Response** - see [Enable configuration for Keycloak]({% link process-services/2.4/config/authenticate.md %}#enable-configuration-for-keycloak) for details.|
```

Comment:
```suggestion
|keycloak.resource| *Required.* The **Client ID** for the client created within your realm that points to Process Services. <br><br>**Note:** For Keycloak compatibility, you must enable **Exclude Issuer from Authentication Response** - see [Enable configuration for Keycloak]({% link process-services/2.4/config/authenticate.md %}#enable-configuration-for-keycloak) for details.|
```

Reply:
The `|` character in line 47 belongs at the end of line 46. If you look at the generated docs page, it appears as a blank line. If you look at the GitHub preview, the table formatting breaks.